src/main/java/hudson/remoting/Channel.java

```diff
@@ -1283,32 +1317,40 @@ public synchronized void close() throws IOException {
      *
      * @since 2.8
      */
-    public synchronized void close(@CheckForNull Throwable diagnosis) throws IOException {
+    public void close(@CheckForNull Throwable diagnosis) throws IOException {
         if(outClosed!=null)  return;  // already closed
-
-        try {
```

Comment:
[diff for reviewers](https://github.com/jenkinsci/remoting/pull/175/files?w=1#diff-8a0e2be489e004b87cc21e1fc8d17d3dR1320)